frontend/src/components/table/TableItem.js

```diff
@@ -715,7 +715,7 @@ TableItem.propTypes = {
   tabIndex: PropTypes.number,
   entity: PropTypes.string,
   getSizeClass: PropTypes.func,
-  colspan: PropTypes.string,
+  colspan: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
```

Comment:
this should be fixed when we refactor TableItem.
We cannot allow having a prop which is sometimes bool and sometimes string.
Please add a FIXME/TODO marker to be handled later and not forget.

Reply:
ok. thanks